packages/page-accounts/src/modals/Create.tsx

```diff
@@ -420,7 +421,7 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
                     )}
                   </Modal.Column>
                   <Modal.Column>
-                    <p>{t<string>('The derivation path allows you to create different accounts from the same base mnemonic.')}</p>
+                    <p>{t<string>(pairType === 'ethereum' && seedType === 'raw' ? 'The derivation path is only relevant when deriving keys from a mnemonic.' : 'The derivation path allows you to create different accounts from the same base mnemonic.')}</p>
```

Comment:
Same comment as above with the conditional outside of the `t` (Otherwise the string ends up non-translatable)